### ISSUE TITLE
Is It Possible to Resume Training Via CLI in Spacy v3 (transformers)?

### ISSUE THREAD

**Issue by John-8704** [explosion/spaCy#8174]
I was training a ner model with a config file and train and validation set , I trained for 10 epochs and model-best and model-last got saved and I use those model for inference.

`python -m spacy train config.cfg --paths.train ./train.spacy --paths.dev ./valid.spacy --output models --gpu-id 0`

Is there any way I can resume training from 10th epoch and continue on again.
